examples/treasure-hunt/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Flask==2.0.1
```

Comment:
Pin a version of werkzeug or use a newer Flask. The newest werkzeug is incompatible with this version of Flask.

Reply:
Added.